doc/POST_INSTALL.md

```diff
@@ -6,4 +6,4 @@ Your video files are stored by default in `/home/yunohost.app/tube/videos`.
 
```

Comment:
I feel like it's probably more relevant to have this as an `ADMIN.md` ? (And also we probably can get rid of the "Configuration" title